src/client/nativeCommunication.ts

```diff
@@ -33,10 +34,12 @@ function getAdSlots(): AdSlot[] {
     return Array.from(advertSlots).map(adSlot => {
         const slotPosition = adSlot.getBoundingClientRect();
         return new AdSlot({
-            x: slotPosition.left + scrollLeft,
-            y: slotPosition.top + scrollTop,
-            width: slotPosition.width,
-            height: slotPosition.height,
+            rect: new Rect({
```

Comment:
Does using the constructor do a different thing to this? https://github.com/guardian/apps-rendering/pull/427/commits/76f44e77ad1d71a5be230399a57d452be866634a